cocoa/src/foundation.rs

```diff
@@ -701,6 +701,336 @@ impl NSRunLoop for id {
     }
 }
 
+pub enum NSURLBookmarkCreationOptions {
+    NSURLBookmarkCreationPreferFileIDResolution = (1 << 8),
+    NSURLBookmarkCreationMinimalBookmark = (1 << 9),
+    NSURLBookmarkCreationSuitableForBookmarkFile = (1 << 10),
+    NSURLBookmarkCreationWithSecurityScope = (1 << 11),
+    NSURLBookmarkCreationSecurityScopeAllowOnlyReadAccess = (1 << 12),
+}
+
+pub enum NSURLBookmarkResolutionOptions {
```

Comment:
Is the the correct way to define `NSURLBookmarkResolutionOptions` and `NSURLBookmarkCreationOptions`?

They're `NSUInteger` types, I'm not sure if this is the right way to define them...

For reference:

* https://developer.apple.com/documentation/foundation/nsurlbookmarkresolutionoptions?language=objc
* https://developer.apple.com/documentation/foundation/nsurlbookmarkcreationoptions?language=objc

Reply:
It would be better to define these as bitflags; this would make it easier to combine them per https://developer.apple.com/documentation/foundation/nsurlbookmarkresolutionoptions?language=objc

Reply:
Sorry! I must have missed this notification.

They're defined as `bitflags!` now, so we should be all good. :smile: